### ISSUE TITLE
Swap Instant Build count with Instant Repair count enhancement

### ISSUE THREAD

**Issue by Federer343** [yuyuvn/KanColleViewer#69]
I've recently swapped from Yuubari's version of KCV to this one, and for the most part I like this one a lot more. The only complaint I have is that on the top bar it lists the number of Instant Builds you have instead of the number of Instant Repairs you have. To me, it seems like Instant Repairs are a much more important resource and would be a better thing to display.

**Comment by c933103** [comment]
The consideration seem to be that you can see no. of bucket in homeport screen, but not burner

**Comment by Federer343** [comment]
@c933103 Yeah I figured that was the function of it, but I still don't think Instant Builds are anywhere near as important as Buckets.  Maybe I'm just too used to Yuubari's version so I'm biased, but it still seems more logical to me.

**Comment by gakada** [comment]
We can add it here like this:

![69](https://cloud.githubusercontent.com/assets/13043588/9101553/5f79ec30-3c23-11e5-907f-d99537e2b0f3.png)

**Comment by Federer343** [comment]
@gakada I mean yeah that would also be a great option.  I guess I just kind of went straight to replacing, not thinking about just adding it on instead.  Your idea is better, lets go with that.

**Comment by artoodetoo3301** [comment]
Sorry, but how do you add that?

**Comment by gakada** [comment]
Since @yuyuvn said that ver.3.8.2r557 is the last release, I will be pushing this and other stuff into my fork's master.

**Comment by yuyuvn** [comment]
thank @gakada
